### ISSUE TITLE
ForEach() with index parameter won't work

### ISSUE THREAD

**Issue by hepspl** [skiptools/skip-lib#3]
When trying to use ForEach(...) with index parameter, eg.:
`
ForEach(Array(options.enumerated()), id: \.element) { index, option in
   ....
}
`

Building fails:
> Cannot access 'element': it is internal in 'skip.lib'

The same happens when `\.offset` is used instead of `\.element`
Is there a workaround?

**Comment by aabewhite** [comment]
Sorry for the trouble. I'll make sure to fix this today. In the meantime I know it sounds strange, but I believe that using .offset and .element in a dummy loop somewhere will also make your ForEach code start working. I.e. somewhere in your module, temporarily add code like the following:

```
let a = [0]
for e in a.enumerated() {
   let _ = e.offset
   let _ = e.element
}
```

As I said, though, I'll make sure to create a proper fix today

**Comment by aabewhite** [comment]
This should be fixed now in Skip 0.8.45. Use Xcode's File->Packages->Update to Latest to get the fixed version. Please let us know if you have any issues, and thanks again for the report.

**Comment by hepspl** [comment]
Works like a charm, thank you
